### PR TITLE
Neurotoxin Drink Balance Take 2

### PR DESCRIPTION
**Pull request by Aurorablade**
((This PR was orginally by Fox))
From Orignal PR:

Nerfs Neurotoxin.

This really should have been done a long time ago. This is nothing more than a nigh-uncounterable infinitely reproducible instant knock-out chemical; it outclasses and generally outperforms even the ultra-high end traitor-only knockout chems.

Neurotoxin now stuns after 55 cycles instead of being an instantaneous knockout. at 12 cycles it makes dizzy, 36 its a weaken.

From New PR:
I will take feed back in direct numbers on what to change and a reason why it should be changed to that number. Any feedback invovleing chemistry or not constructive regarding numbers will not be taken into account unless I see its really good.

:cl: Fox McCloud and Fethas
tweak: Neurotoxin knockout has a very long delay as opposed to being instant (fox)
tweak: Ether now has a 10 percent chance to weaken between 1-15 cycles (Fethas)
/ :cl:

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
🤔 Has Fethas gone.. _nurotic_?

**Comment by Lady-Luck** [comment]
I'll post the diagram again.

http://i.imgur.com/2ELGCpi.png

I'm tired of stuff getting nerfed into the ground. I warned everyone it's what would happen if the genetics PR went through. And while this doesn't affect all of chemistry, it certainly means that everyone's just going to puck up a stunprod from now on.

How about a total pass on nonlethal chemicals instead of focusing on just one thing?

**Comment by Lady-Luck** [comment]
The cycle count needs to be WAY lower. If you are only focusing on neuro-toxin this PR, then it should remain way better than all the other nonlethal chems, because they are quite frankly awful.

The entirety of nonlethal chems needs a balance pass, obviously.

**Comment by FreeStylaLT** [comment]
I at the moment don't see much purpose in having an instant, or anything near it, knock-out chem at all. Though, I may be missing some aspects of its purpose. I'd like people to fill me in on why they think it needs to be lower / changed differently.

**Comment by Lady-Luck** [comment]
>I at the moment don't see much purpose in having an instant, or anything near it, knock-out chem at all. Though, I may be missing some aspects of its purpose. I'd like people to fill me in on why they think it needs to be lower / changed differently.

It's a similar argument to what Fox used about not nerfing cryo: Look at what it's competing against. A stun prod is easy to make, it's instant, and it stuns for a reliable amount of time. Neuro-toxin is one of the few good alternatives.

Every other nonlethal chemical is completely terrible. They take a long time to work and don't even work reliably. Even the traitor nonlethal chems are damn near worthless.

What this PR does currently (as well as the previous PR) is it removes neuro-toxin from the game, effectively. And I don't like things getting nerfed so hard you might as well have just removed it.

**Comment by FreeStylaLT** [comment]
You claim it is an alternative to a stunprod and that every other non-lethal chemical is completely terrible. I assume you speak of ones that incapacitate / stun a player.

You also seem to show a sense of concern when it comes to making Neurotoxin on par to those other chems, in turn making it no longer a viable alternative to a stunprod.

I ask, then, why _must_ there be chem-based alternatives to stunprods in terms of effectiveness?

**Comment by Draconic-saint** [comment]
I'd be up for removing this version of neurotoxin in it's entirety. Or it's active parts except for the alcohol. 

"Offensive" chems are a bitch to balance between OP and shit. But this thing should be neither.
It's a goddamn cocktail. For me ideally this would either be reworked and reclassified as a purely chem-based thing, that takes time and resources to make.

A cocktail's secondary effects shouldn't be able to be used as weapons.
 
As much as I understand that it's not fun to see things removed from the game, sometimes it's necessary.
I feel like this is one of those times.

**Comment by Lady-Luck** [comment]
>I ask, then, why must there be chem-based alternatives to stunprods in terms of effectiveness?

Well of course there doesn't have to be, but I'm tired of 'nerfs' being masked attempts at removal. Nerfing into oblivion is just as bad as removal if not worse, but it's much easier to pass.

And I believe that non-lethal chemicals should have a place in the game. There aren't enough non-lethal solutions in the game as it is.

**Comment by scrubmcnoob** [comment]
why not try modifying the PR in hopes to change people's minds unlike the last one? a legit copy pasta is not going go far if the last one just got overly negative feedback. try finding a compromise my friend

**Comment by Lady-Luck** [comment]
@scrubmcnoob brings up a good point. Every single criticism the last PR had is still relevant and this is just hitting the reset button. This isn't even a new PR.

This one should be closed for being a carbon copy and the original reopened.

**Comment by Aurorablade** [comment]
The only part not copy pasted in the desc is this line.  I will take feed back in direct numbers on what to change and a reason why it should be changed to that number. Any feedback invovleing chemistry or not constructive regarding numbers will not be taken into account unless I see its really good.

**Comment by Lady-Luck** [comment]
>The only part not copy pasted in the desc is this line. I will take feed back in direct numbers on what to change and a reason why it should be changed to that number. Any feedback invovleing chemistry or not constructive regarding numbers will not be taken into account unless I see its really good.

The feedback doesn't involve numbers because it is part of a larger problem that has nothing to do with numbers. You can't just draw a line in the sand that only benefits you.

**Comment by Lady-Luck** [comment]
Great feedback from @Bigtimetripod from the last PR:

Oke... This bothers me... Yes Neuro-tox is kinda OP, but its the only knock-out chem you can relay on. All other chems take atleast 15-30 seconds to have effect. In a fight or trying to capture someone that time is enough to do all kinds of things. This even gives enough time for the victim to retaliate and kill you if robust enough. What help is a knock-out drug that just get you killed? I usually use Neuro-tox for kinda as a staged knock-out mix. Have Neuro-tox knock-out and give enough time for some other chemical to take effect. I would say to leave it since its the only chem that even if smart enough, a civi can make. Yes it hard to make it as a civi and bot get caught, but its possible. For Scientist or medical doctor is easy.

Now for the other topic, poison bottles. Lets just start by sayin that not every traitor gets em. Poision bottle is only obtainable by Research Director , Chief Medical Officer, Medical Doctor, Psychiatrist, Paramedic, Virologist, Bartender. Even if you try and get the chemical you want. Its REALLY REALLY RNG. You can get a multiple bottles of the same stuff, cost is too high and for 2 tc you can get two clips of ammo for the pistol. Maybe make the poison bottle for everyone and make a subcategory when buying the bottle.

Knock-out chems 2-3 TC
Damage chems 3-5 TC
Lethal chems 5-8 TC

Have something like Pancuronium and Sodium Thiopental as knock-outs. Damage chems maybe Sarin and Coniine. Lethal chems Initropidril, Prions or nanomachines. The poison bottles are the rarest to see traitor take just because of their high cost and randomness. They are not worth it. Since they are also in limited supply, there is no point to take em. 4 bottles of random or e-mag with unlimited and much more uses? 3 bottles or gun with 2 mags? 6 bottles of random and maybe same useless stuff for a crossbow with unlimited stuns and guaranteed knock-down pluss toxin damage.

Personal opinion? Chems are being hugboxxed... Still havent got over that Phlogiston doesnt ignite people anymore and now this... Whats next? Omnizine nerf? Oh wait... it already was... SGS gets removed? Perfluorodecalin no longer mutes? Why not gives Stun batons stun delay? Stun baton + Bluespace battery = Stunlock forever basically

**Comment by Aurorablade** [comment]
I can and I will draw a line in the sand. Chem balancing can be discussed elsewhere but not on a pr about a drink. Its not a chem. Its a drink. You give me feed back on how else to tweak.it so its.balanced I will modify.

**Comment by Lady-Luck** [comment]
This is absurd. Every single point from the previous PR is still valid.

@SamHPurp @Kyep @Doukhan @Davdichan @BiberDark @uraniummeltdown @Dopparn @fludd12 @Natje0 @Mijhead

**Comment by Aurorablade** [comment]
To parrot what jedr said "A cocktail's secondary effects shouldn't be able to be used as weapons." I am open to modifying it if provided something g good

**Comment by scrubmcnoob** [comment]
I'm pretty sure you got sufficient feedback? It seems people are clearly unhappy with this current PR, as it stands, and a re-scaling of values couldn't hurt.
Either way we look at it, a downscale of neurotoxins is still a downscale at the end of the day.
And this kind of looks like a meme pr when you copy pasta down to the change log of a PR that was literally just closed due to negative feedback. Take it constructively and find a good resolution.

**Comment by FreeStylaLT** [comment]
As far as I am concerned, I do not see any reason why a bartender's drink should rival a stunprod and be able to be weaponized. It seems like a joke taken to extreme levels of abuse in certain cases.

An alternative to a stunprod should in all likelihood exist, but it should not be a chem, but rather something just as available to everyone as the stunprod, in a best case scenario.

Considering that no one provided any valid reasoning as to why a chem-based alternative to the stunprod _should_ exist, I do not care how this gets nerfed, as long as it is made inferior to the stunprod, if that is the main comparison / balance / arguing point.

**Comment by Lady-Luck** [comment]
>As far as I am concerned, I do not see any reason why a bartender's drink should rival a stunprod and be able to be weaponized. It seems like a joke taken to extreme levels of abuse in certain cases.
>
>An alternative to a stunprod should in all likelihood exist, but it should not be a chem, but rather something just as available to everyone as the stunprod, in a best case scenario.
>
>Considering that no one provided any valid reasoning as to why a chem-based alternative to the stunprod should exist, I do not care how this gets nerfed, as long as it is made inferior to the stunprod, if that is the main comparison / balance / arguing point.

You do realize that moving the playing ground to a PR where the huge pile of negative feedback to a fresh PR after everyone already moved on is a very deceptive practice, right? The github has a bad reputation for shady tactics like this.

**Comment by FreeStylaLT** [comment]
The past PR did not answer my question, either, otherwise, I would not be asking it here.

On top of that, I can attest that the PR was made not in order to dodge feedback, but because it felt like Fox closed his pre-emptively, Fox could have re-opened the PR, but Fethas got to making this one first.

**Comment by Lady-Luck** [comment]
I already answered your question. There is already a lacking in non-lethal solutions in the game and removing another isn't desirable. 

Asking why something _should_ exist is an extremely broad question that could be applied to almost anything. Why _should_ IPCs exist? They create a ton of code snowflake and have been notoriously difficult to maintain. Why should anything exist?

It's good to have options. Nerfing Neuro-toxin removes an option without really putting anything in its place. If you feel it's silly for it to be a drink, then give that same status effect to a different chem or make a new one. Clearly it's the status effect that's the problem here and the source is really secondary.

**Comment by Spacemanspark** [comment]
IPCs should exist because they bring something interesting to the table.
This does not.

**Comment by Lady-Luck** [comment]
>IPCs should exist because they bring something interesting to the table.
This does not.

That is completely subjective. Nonlethal solutions allow for a lot of interesting things to happen, and I think it's a hell of a lot more interesting than just making a stunprod.

**Comment by Kyep** [comment]
Four reasons why a stun chem should exist:
- Variety. Stuns are a critical part of SS13 combat. Forcing everyone except security to rely on a single weapon (stunprod) to do them removes possibilities from the game without adding anything.
- Interactivity. To make neurotoxin, you have to get PGGB from the bar, a gun from R&D/medbay, and make the ether in a chem set. You have to go to several places and interact with multiple people. To make a stunprod, all you have to do is pick up the parts from maint and assemble it. You never have to interact with anyone, or do anything complicated. Having neurotoxin in the game encourages people to travel around, interact with others, etc, rather than just making their gear alone in maint.
- Balance. Neurotox is perhaps the only viable ranged stun weapon that isn't locked to either antags or security. Removing/nerfing it changes the balance of power between security and everyone else specifically. And no, not in a way you'd like! Security will now be more hassled than ever, as it is no longer possible for medbay or science to use neurotoxin to stun and eject greytiders/troublemakers from their departments. Now, they have to call security to deal with it - which increases the burden on security for no substantial benefit. These people aren't going to get more than 5 minutes, it isn't worth processing them. Without departments having the ability to defend themselves from random greytide, we'll see more random greytide.
- Community Approval. The previous PR to remove neurotoxin got 10 votes against, and only 1 vote in favor. Nothing that has a 1:10 for/against ratio should get merged. Heck, I've seen things rejected because they had a less than 1:1 for/against ratio, yet its OK for this change to have a 1:10 ratio? What? If you force it through even against overwhelming opposition like this, it will make people lose faith in the ability for maintainers/etc to listen to the community. You may have good reasons to want to rebalance it. I understand and even agree with some of them. However, forcing it through despite widespread opposition is a really short-sighted way of handling the topic.

**Comment by FreeStylaLT** [comment]
That's what I was looking for! Unfortunately, I can't address those points right now, but that's a much better way of looking at it than what I've seen in either PR so far!

**Comment by SamHPurp** [comment]
@Lady-Luck thanks for the Ping.

Yeah... this is still bad for all the reasons already stated on this PR.

Also, what do you hope to achieve by raising mock PRs designed to belittle those with opposing views @Fox-McCloud ?

**Comment by Kyep** [comment]
The last question I got from someone looking to make a neurotox gun, was from.... a player who plays CMO a lot. He wanted a way to get greytide and other problematic individuals out of medbay without hurting them, and without it turning into a fight in which his staff got hurt. In theory, he could ask security to come and remove them, but in practice that never works, as security always has something better to do than deal with trespass/greytide in Medbay. Even if they do come, usually the perp has left by that point. Since learning to make neurotox, this CMO is now able to deal with minor issues like this in Medbay himself. Tranq them and throw them out. This sort of immediate consequence has saved Medbay from much greytiding, and made him a lot more willing to play CMO, because he feels able to actually deal with these minor problems. All of Medbay has benefited from this. 

I don't know if this guy would continue playing CMO without neurotox. It is, AFAIK, the only non-lethal solution the guy has found thus far to the issue of trespass and greytide in Medbay. It being in the game makes a major difference to his ability to actually control his department.

**Comment by Draconic-saint** [comment]
Just because traitor chems and science chems are hilariously shit, doesn't mean neuro-toxin, the drink, in it's current stage is OK. It's very much not.

Does it get knocked the down from it's OP "memechem" pedestal? Yes, and rightfully so. A goddamn **drink** shouldn't be on that level. Because that's what this was intended to be. A drink. It's literally classified as that in the code. Instead we have gutted our chem-system so badly this drink now trumps all chems on the robust scale. 

We all want fun and fair chemicals. If this, in it's current state, is your opinion off either, i'm gonna start calling you out on your powergaming shit. 

And about the non-sec needing it to keep the greytide in check: this should not be their only option. Hell this should not even be on their priority list unless they're playing sec. If a CMO worries more about keeping the tide out of his bay than actually running it, that's an issue we need to solve another way, but not by shielding this from nerfs.

TL;DR: just because just about every toxin or poison deserves a buff and it's the last stunchem, doesn't mean this should get a "get out of jail free" card for nerfs.

**Comment by Lady-Luck** [comment]
>EDIT: @Kyep I read up a bit more, yes a stunchem should exist, but this one should not be it.

That is perfectly fine, but I am against making such a change until that chemical is introduced and not before.

**Comment by Spacemanspark** [comment]
Ether :^)

**Comment by Draconic-saint** [comment]
I wholeheartedly disagree, as my TLDR summarises. It's a good change, that can be followed up by other good changes.

Hell it might just get rid of the old "meta" and make it so we have a clean drawing board again to rebalance all chems at the same time, without having the risk of falling into the trap of using neuro-toxin as a control standard.

**Comment by Lady-Luck** [comment]
>Ether :^)

It's weak, it takes a long time to work, and is inconsistent. This also isn't a stun chem. This is obviously just a troll reply and I'd appreciate actually giving constructive feedback.

>Hell it might just get rid of the old "meta" and make it so we have a clean drawing board again to rebalance all chems at the same time, without having the risk of falling into the trap of using neuro-toxin as a control standard.

No one wants neuro-toxin to be used as a control standard. EVERYONE agrees that it's too good. The issue is not having a chem to fulfill the role of neuro-toxin once it is removed.

If you remove it, and approach this subject later, your 'control standard' is all the other non-lethal chemicals which are just all sorts of terrible.

**Comment by Spacemanspark** [comment]
>This is obviously just a troll reply

Actually no I was being serious

**Comment by Lady-Luck** [comment]
>Actually no I was being serious

It's not a stun chem. You have to use it in quantities that would remove a player from the game for much longer than would be desired if you want to use it at all, and even when you do, it still sucks because of how reliable it is.

**Comment by Draconic-saint** [comment]
If it exists, it'll be hard to ignore when drawing up the balance for the rest. That's what I think atleast. If everything is equally shit, I would hope that that resulst in a better and more equal division of power and roles.

**Comment by Kyep** [comment]
I think everyone admits that neurotoxin is, to some degree, too good, relative to everything else.

The problem is: there seems to be this obsession with nerfing/removing everything that is good/useful, without making any attempt to fix ether, or traitor chems, or anything else that is underpowered by comparison.

Its like going through a parking lot, and systematically destroying every good car, because its 'too good', whilst never fixing the ones that have problems. You use the fact that there are broken cars to argue that the good cars should be destroyed because its 'unfair' they're so good in comparison... but you never fix the broken ones. You just repeat this argument regularly, until all the good cars have been destroyed, then you look out over your lot of junkers, proud to have 'balance' by having made everything shit.

Also, the fact it is a 'bar drink' is irrelevant. Totally irrelevant. Tons of things were created for one purpose, then they found actual *usefulness* doing something else. Some of the greatest breakthroughs, producing some of the most useful things in human history, were complete accidents. Heck, even stuff like penicillin. It was a mould. Before it was discovered to be actually useful as a life-saving anti-biotic. I feel like if we were having this conversation about penicillin, people would be arguing against its use because "a mold shouldn't be that strong". What something was originally created for is irrelevant. What matters is what it is useful for today. Looking at neurotoxin and saying its too strong for a bar drink, is like looking at penicillin and saying its too strong for a mold. You're right that its stronger than the other things in its class, but to say its too strong on that basis alone is rather like being presented with a golden statue as a gift and saying you don't like it because statues aren't meant to be that impressive. For god's sake, its USEFUL, which is more than you can say for most things. Don't hate it just because its actually good for something.

**Comment by Draconic-saint** [comment]
You think of the car park argument, but I think of letting Messi play with my local fifth-grade team football team. That shit just isn't fair. Sure messi is good. And he has his place. But NOT there.

I would love to see that guy play in a competitive environment, makes for good entertainment. but when there's literally 0 opposition, it just becomes sad.

Just like that, a stunchem is good, and has it's place. But not as a bardrink. And i'm talking about how it's ICly viewed. I'm talking about how it's classified. It's called a drink, when clearly, it has potency and effects **GIVEN** to it far b(e)yond what is usually the norm for a drink.

This isn't something that was discovered. This was made. All components and effects were known.
Flemming didn't **know** there was an anti-biotic component in that moss.

I don't want to see this nerfed for what it does, I want it nerfed for how and where.

**Comment by Mijhead** [comment]
Ich bin immer noch tot, nerf bitte meine Fraulein

**Comment by Rumiluntti** [comment]
People are actually defending instant stunchems that are piss easy to make?

**Comment by Spacemanspark** [comment]
Welcome to the world of validhunting.

**Comment by Bigtimetripod** [comment]
Its being defended cause its the last of its kind. PETA exists for a reason and seeing em Nerf PRs just angers the people and they want to keep their alternative to stunprods. 

Wild suggestion!

Lets remove sec instastun tasers. Ranged stun for ranged stun. Equal balance of power will be restored between sec and rest of the station. 

Dammit... maybe i should make my first PR balancing all the stun chems... since i do got a lot of experience with em, would only be fair. All other "stun chems" just will not cut it. And why people keep sayin ether? Its the worst out there! Not a proper stun, but a faint fest like sulfonal and sulfonal starts workin faster then ether...

**Comment by Davdichan** [comment]
@Rumiluntti to imply a stun prod is difficult? The only reason Neuro-toxin looks OP because the chems that are supposed to compete with it are weak as shit and lack any reliable stealth application outside of spiking food and drinks which is exactly the niche Neuro-toxin should fill.

Traitor chems and other 'combat' or stun chems need to be reworked to be made viable or given alternative delivery methods so the target isn't immediately aware they were drugged. Nerfing the only useful nonlethal-pacification-chem we have into the dirt isn't going to fix anything.

**Comment by mrturkeytoe** [comment]
I agree that neuro-toxin should be given a nerf of some sort, whether that's making it harder to mix or be less effective.

HOWEVER, the biggest reason it seems so OP is because every other option besides stun-prods has been nerfed into the ground or was shit to begin with.

Ether? It's a joke.
Morphine? It's easier to kill them with it than stun them.
Sulfonal? It's /okay/ but it also kills them.

Those three and Neuro-Toxin are the only stun chems, and Neuro-Toxin is the only one anyone would even notice if it was removed.

I suggest that you make one big PR that re-balances chemistry as a whole so everything is balanced acceptably.

ALSO BTW this isn't the first PR that has nerfed chemistry rather than buffing things it has to compete with [#5251](https://github.com/ParadiseSS13/Paradise/pull/5251)

**Comment by Aurorablade** [comment]
ether, high availability, normal depletion, infinite reproducibility, but long impact time with a 17 percent chance to knock them out early.
neurotoxin the NON-DRINK, high availability, very very high depletion rate, infinite reproducibility, very quick impact time.
and pancuronium,l ow availability, low depletion rate, limited amount on the map, quick impact time.

**Comment by Draconic-saint** [comment]
Tripod please, comparing this to endangered animals is hilariously bad an argument. 

This is a piece of code, easily fabricated, changed and honestly a non-issue, compared to real issues. You paint yourself as a defender of something endangered, but you seem to forget alternatives can justr as easily be coded in as that this shit can be nerfes

**Comment by Aurorablade** [comment]
Experiment: cylcles 1 to 15: 10 percent chance to weaken 3 for ether. Can be reverted or or numbers tweaked.
Basis:The feeling i always have when i have to take nyquil or bendryl...(i know different drugs but that i am trying to use that as the feeling of it..shut up.)

**Comment by LightFire53** [comment]
Wow. People are angry that their easy to make bar drink won't be an instant stun and continously KO chemical?

If your argument is to have instant stun chems, fine. Make that argument on a CHEMICAL PULL REQUEST. It's a bar drink. It is silly that booze and a little bit of a chemical (that takes time to KO you, as our lovely Lady Luck has pointed out at least once) can instantly hinder and incapacitate you for a couple of minutes whIle you slowly purge it from your body lying on the bar floor.

Out of all of our options, this one is my personal preference. Still does the job, just not instant.

**Comment by Lady-Luck** [comment]
>Wow. People are angry that their easy to make bar drink won't be an instant stun and continously KO chemical?

I think you haven't read enough feedback if this is the conclusion you came away with. Everyone agrees its current state is too good.

**Comment by Vivalas** [comment]
I need to read through this drama some other time, but in the meantime this should become a new family of memes.

"Two kinds of people in the world: Neurotoxin, or Stunprod."
"Oh god, a neurotoxin PR." (Repeatedly proposing a contentious topic)
"Let's balance the bad stuff by making the good stuff shit."

Except the last one isn't a meme and it's a trend people are hellbent on bringing this once unique and interesting codebase down with.

**Comment by Fox-McCloud** [comment]
Going to close this--negative response.
